### ISSUE TITLE
dev20220311: debug log: 'drawing at page xxx ' does not match display content

### ISSUE THREAD

**Issue by W-Geronius** [norbert-walter/esp32-nmea2000-obp60#30]
![image](https://user-images.githubusercontent.com/18160302/157947976-5fad358d-157c-4b7e-944a-c6796a39e343.png)

this situation came up, when I briefly stopped (and reestablished) WiFi connection to NMEA0183 provider (SignalK). The MFD display is correct, the debug however shows weird values ...

**Comment by norbert-walter** [comment]
Yes, thats right. Internal are all data present but we using a valid flag and setting the flag by missing values. I think it is uncritical.

**Comment by W-Geronius** [comment]
uncritical, even useful! 
If missing data flag is added to the debug log entry, no additional documentation would be required
